data/strings/strings.txt

```diff
@@ -11514,8 +11514,8 @@
     nb = Åpen
     nl = Open
     pl = Otwarte
-    pt = Aberto
-    pt-BR = Aberto
+    pt = Abre
+    pt-BR = Abre
```

Comment:
It's better to remove equal pt-BR strings if they are equal to pt. Our twine should properly handle that.

Reply:
Done in all strings in this PR and a couple of related strings